### DISCUSSION TITLE
Moving to SimpleITK 2.1.0

### DISCUSSION THREAD

**Discussion by sarthakpati** [TorchIO-project/torchio#628] [General]
Hey,

Are there any plans to move to the new SimpleITK version? I know that earlier versions had [an issue](https://github.com/SimpleITK/SimpleITK/issues/1239) but this has seen been resolved and a new release has been pushed.

Cheers,
Sarthak

**Comment by fepegar** [comment]
Hi, @sarthakpati. What do you mean by "move" to the new version? Everything should work on 2.1.0.

**Reply by sarthakpati** [reply]
Ah okay, I missed the notation for 2.0 [[ref](https://github.com/fepegar/torchio/blob/master/setup.py#L15)]:
```python
'SimpleITK!=2.0.*'
```

Thanks!
